### ISSUE TITLE
after stacked decoder layer，I think it should be concatenate with a linear and softmax layer.

### ISSUE THREAD

**Issue by flyboyer** [CyberZHG/keras-transformer#5]
after stacked decoder layer，I think it should be concatenate with a linear and softmax layer.but in your code, it's 
`dense_layer = EmbeddingSim(
        trainable=trainable,
        name='Output',
    )([decoded_layer, decoder_embed_weights])`
why do this operate?
